src/styles/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  margin: 0 ;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
+  width: 200px;
+  font-family: "Roboto", sans-serif;
```

Comment:
you can move `font-family: "Roboto"`, sans-serif; to the `html` tag and don't repeat it all over your code